scripts/gh/overview-dag.sh

```
@@ -0,0 +1,143 @@
+
+# Initialize edges file (truncate)
+: >"$tmpdir/edges.txt"
+
+# Batch fetch open issue bodies via GraphQL to avoid N+1 API calls
+owner="${REPO%%/*}"; name="${REPO##*/}"
+# Paginate through open issues (max 100 per page)
+echo "[]" > "$tmpdir/all_nodes.json"
+after=""
+while :; do
+  if [[ -z "$after" ]]; then
+    query='query($owner:String!,$name:String!){ repository(owner:$owner,name:$name){ issues(first:100, states:OPEN, orderBy:{field:CREATED_AT, direction:DESC}){ pageInfo{ hasNextPage endCursor } nodes { number title body } } } }'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)